front-end/src/components/Markdown.tsx

```diff
@@ -102,6 +104,9 @@ export const MarkdownTreeNode: FunctionComponent<{
   if (parseTreeNode.kind === "table") {
     return <DrawTable table={parseTreeNode.child} />
   }
+  if (parseTreeNode.kind === "list") {
+    return <DrawList list={parseTreeNode.child} />
```

Comment:
why is this needed? Lists can be included using standard markdown syntax:
```
- item a
- item b
```

Reply:
It can be nested List and instead of `-` it uses a list point

Reply:
Nested lists should also use standard markdown syntax:
```
- item a
  - subitem 1
  - subitem 2
- item b
```

I don't think we need to allow the question generator to change the style of the list. If you prefer list points in general, change this globally.

Reply:
But we currently don't have lists in Markdown. This would be just rendered as normal Markdown text and the spaces in front of `subitem 1` and line breaks would be removed. 

With this we could create a list block. Space to the top and bottom and always the same style  

Or am I missing something?

Reply:
- My point is that there is standard syntax for lists in Markdown, which we should use instead of a `list` code-block. Yes, this requires finding the correct regex etc.
- We are slowly reaching a level of complexity for our parser where we may just want to use a standard markdown parser such as https://marked.js.org instead of rolling our own. Probably @AbcAeffchen will object here?
- If you want, you can decide to keep the quick solution using a code block, and put the proper list syntax to the someday/maybe todo list.

Reply:
Yes I understand what you mean. 

https://www.markdownguide.org/basic-syntax/#unordered-lists
There are 3 different types for unordered Markdown lists. I took the list with the `-`, maybe not the same regex but mainly it should do the same. It parses the list-string line by line into a string tree.

Reply:
I would like to keep the quick solution and add an issue once this PR has been merged, so we don't forget it